manifests/plugin/threshold.pp

```diff
@@ -2,12 +2,16 @@
 class collectd::plugin::threshold (
   $ensure   = 'present',
   $interval = undef,
+  Array[Collectd::Threshold::Type]   $types   = [],
```

Comment:
can you please align the `=` chars please?

Reply:
Done in dd7fb6d4fc8bb72301fd4ed80d894b787931f3e1